### ISSUE TITLE
Standalone Low Frame Rate Dialog - Can we disable it?

### ISSUE THREAD

**Issue by michaeloldman** [TheGameCreators/GameGuruRepo#397]
Hi all :)

Can we disable the 'Low Frame Rate Detected' dialog in standalone games. In my book it really is quite pointless :) 

I have been messing around with building a standalone of one of my WIP levels today and got this dialog saying 'Low frame rate detected .....' As it happened while a drone was intent on destroying me it caused me to loose a life because the dialog hogged the screen, but didn't freeze game play.

As it happens there was no noticeable frame rate drop for me at all, the level was playing just fine.

I don't think we need a dialog to tell us when frame rates are low anyhow, as it would be obvious, so can we disable this thing, or better still just get rid of it?

As a side note the standalone was pretty much all custom media, and aside from the _illumination maps not getting copied over, and of course the drones mostly being real dippy, it worked a treat, even copied all the media for the second level as well even though I didn't run it before building the standalone.

Reliquia....

**Comment by LeeBamberTGC** [comment]
Yes you can find this in the SETUP.INI file, called "hidelowfpswarning". Feel free to close this if it answers the issue, thanks.
